plugins/com.google.cloud.tools.eclipse.usagetracker/src/com/google/cloud/tools/eclipse/usagetracker/AnalyticsPingManager.java

```diff
@@ -275,7 +274,7 @@ static String getParametersString(Map<String, String> parametersMap) {
       try {
         resultBuilder.append(URLEncoder.encode(entry.getValue(), "UTF-8"));
       } catch (UnsupportedEncodingException uee) {
-        throw new RuntimeException("UTF-8 not supported");
+        // JVM implementations not supporting UTF-8 are not expected.
```

Comment:
you need to restore the RuntimeException if you're not going to switch to a more reliable encoder here